### PR TITLE
refactor: remove ngbNav from storefinder library

### PR DESCRIPTION
**Pull request by Matejk00**
Closes: CXSPA-1260

### CODE REVIEW

File: feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html [hunk 1]
```diff
@@ -63,66 +63,89 @@
...
-            {{ 'storeFinder.listView' | cxTranslate }}
-          </a>
-          <ng-template ngbNavContent>
-            <div class="cx-address-col">
-              <div class="cx-store-details" *ngIf="isDetailsModeVisible">
-                <cx-store-finder-store-description
-                  [location]="storeDetails"
-                  [disableMap]="true"
-                ></cx-store-finder-store-description>
+      <ul role="tablist" class="nav">
```
Comment (morganm58) [inline feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html]: We should add the `cx-` prefix to the nav class

File: feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html [hunk 2]
```diff
@@ -63,66 +63,89 @@
...
+        <li class="nav-item" *ngFor="let mode of displayModes | keyvalue">
```
Comment (morganm58) [inline feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html]: We should add the `cx-` prefix to the nav-item class

File: feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html (hunk 1 above)
Comment (Matejk00) [reply feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html]: @morganm58 this class depends on bootstrap styling and is provided by the bootstrap library. 
If we add such a prefix, all default styling will gone.

It's a similar usage like `btn` or `btn-primary` for example.

File: feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html (hunk 2 above)
Comment (Matejk00) [reply feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html]: Same case as above.

File: feature-libs/storefinder/styles/components/_store-finder-list.scss [hunk 3]
```diff
@@ -23,6 +23,13 @@
     .cx-map-col {
       height: 70vh;
     }
+
+    .nav-item {
```
Comment (morganm58) [inline feature-libs/storefinder/styles/components/_store-finder-list.scss]: we should use a `cx-` class to add the styling here

**Review by psmul — APPROVED** [review]

File: feature-libs/storefinder/styles/components/_store-finder-list.scss (hunk 3 above)
Comment (Matejk00) [reply feature-libs/storefinder/styles/components/_store-finder-list.scss]: done

**Review by morganm58 — APPROVED** [review]

**Pull request merged**